cmds/copy.c

```diff
@@ -115,8 +118,10 @@ static int cmd_parseDev(handler_t *h, addr_t *offs, size_t *sz, cmdarg_t *args,
 static int cmd_copy(char *s)
 {
 	size_t sz[2];
+	ssize_t res;
 	addr_t offs[2];
 	handler_t h[2];
+	const char *file[2];
```

Comment:
Wouldn't it be better to define a structure
e.g.
```
typedef struct{
	size_t sz;
	addr_t offs;
	handler_t h;
	const char *file;
} copy_dev_t;
```
In `cmd_copy()` define the variables
```
copy_dev_t src;
copy_dev_t dst;
```
and then 
```
static ssize_t cmd_cpphfs2phfs(copy_dev_t *src, copy_dev_t *dst)
static int cmd_parseDev(copy_dev_t *dev, cmdarg_t *args, unsigned int *argsID, unsigned int argsc)
```
This will reduce the number of arguments in functions:
`cmd_cpphfs2phfs()` from 6 to 2
`cmd_parseDev()` from 7 to 4

What do you think about it ?

Reply:
@pawelpisarczyk I know that we try to minimalize number of structures and typedefs to the minimum, but it might be justified in above case. In the above code are arrays of 2 elements of members of such proposed structure. Defining it (as struct copyDev or typedef copyDev_t) IMHO would code readability.

Reply:
Yeah, the reason why I decided to create arrays of 2 elements to reduce the instances of additional structures and typedefs as it is one of the our rules. 
I agree that creating structure in this case will increase to code readability.